### DISCUSSION TITLE
[DONE ✅]  Please add email name search to: Server settings: Users  ??

### DISCUSSION THREAD

**Discussion by Eskyee** [btcpayserver/btcpayserver#2271] [Ideas / Feature Requests]
Could we possibly add a email name search to Server settings: Users  
or list Server settings: Users from  A to Z 
as hunting for a customer email address to delete or modify with over 5000 users is very painful. not a great UX
plus being able to delete old users accounts with a list filter, would also be a great feature.

@dennisreimann @bolatovumar 

<img width="1246" alt="Screenshot 2021-02-11 at 09 08 08" src="https://user-images.githubusercontent.com/30862925/107617970-b381b780-6c48-11eb-9ab9-9ac06c0c709b.png">

thanks guys let me know what you think ?? good idea!! cheers

**Comment by bolatovumar** [comment]
@Eskyee sounds like a good idea to me. I will see what I can do.

**Reply by bolatovumar** [reply]
Made a PR here: https://github.com/btcpayserver/btcpayserver/pull/2305
